Tribler/Policies/BoostingManager.py

```diff
@@ -0,0 +1,493 @@
+        lt_torrent = self.session.lm.ltmgr.get_session().find_torrent(ihash)
+        if download and lt_torrent.is_valid():
+            self._logger.info("Writing resume data for %s", str(ihash))
+            torrent['pstate'] = download.write_resume_data()
+            self.session.remove_download(download, hidden=True)
+
+    def _select_torrent(self):
+        """
+        Function to select which torrent in the torrent list will be downloaded in the
+        next iteration. It depends on the source and applied policy
+        """
+        torrents = {}
+        for infohash, torrent in self.torrents.iteritems():
+            # we prioritize archive source
+            if torrent.get('preload', False):
+                if 'download' not in torrent:
+                    self.start_download(torrent)
+                elif torrent['download'].get_status() == DLSTATUS_SEEDING:
+                    self.stop_download(torrent)
+            elif not torrent.get('is_duplicate', False):
+                if torrent.get('enabled', True):
+                    torrents[infohash] = torrent
+
+        if self.settings.policy is not None and torrents:
+            # Determine which torrent to start and which to stop.
+            torrents_start, torrents_stop = self.settings.policy.apply(
+                torrents, self.settings.max_torrents_active)
+            for torrent in torrents_stop:
+                self.stop_download(torrent)
+            for torrent in torrents_start:
+                self.start_download(torrent)
+
+            self._logger.info("Selecting from %s torrents %s start download", len(torrents), len(torrents_start))
+
+    def load_config(self):
+        """
+        load config in file configuration and apply it to manager
+        """
+        validate_source = lambda s: unhexlify(s) if len(s) == 40 and not s.startswith("http") else s
+
+        def _add_sources(values):
+            """
+            adding sources in configuration file
+            """
+            for boosting_source in values:
+                boosting_source = validate_source(boosting_source)
+                self.add_source(boosting_source)
+
+        def _archive_sources(values):
+            """
+            setting archive to sources
+            """
+            for archive_source in values:
+                archive_source = validate_source(archive_source)
+                self.set_archive(archive_source, True)
+
+        def _set_enable_boosting(values, enabled):
+            """
+            set disable/enable source
+            """
+            for boosting_source in values:
+                boosting_source = validate_source(boosting_source)
+                if boosting_source not in self.boosting_sources.keys():
+                    self.add_source(boosting_source)
+                self.boosting_sources[boosting_source].enabled = enabled
+
+        # set policy
+        self.settings.policy = self.session.get_cm_policy(True)(self.session)
```

Comment:
`get_cm_policy(True)` returning a class. Adding `(self.session)` instantiate it.